test/js/samples/reactive-class-optimized/input.svelte

```diff
@@ -0,0 +1,24 @@
+<script context="module">
+	let nonReactiveModuleVar = Math.random();
+	let reactiveModuleVar = Math.random();
+</script>
+
+<script>
+	nonReactiveGlobal = Math.random();
```

Comment:
Great tests already! Could these be enhanced by import statements? One for a non reactive variable, one for a store that is subscribed to and therefore reactive.

Reply:
Done! Added 3 more tests:
- An imported variable that isn't reactive
- A store that's subscribed to
- A reactive declaration